airbyte-integrations/connectors/source-stripe/source_stripe/schemas/checkout_sessions_line_items.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": ["null", "object"],
+  "properties": {
+    "id": {"type": ["null", "string"]},
+    "checkout_session_id": {"type": ["null", "string"]},
+    "object": {"type": ["null", "string"]},
+    "amount_subtotal": {"type": ["null", "integer"]},
+    "amount_total": {"type": ["null", "integer"]},
+    "currency": {"type": ["null", "string"]},
+    "description": {"type": ["null", "string"]},
+    "discounts": {
+      "type": ["null", "array"],
+      "items": {
+        "type": ["null", "object"],
+        "properties": {
+          "amount": {"type": ["null", "integer"]},
+          "discount": {
+            "type": ["null", "object"],
+            "properties": {
+              "id": {"type": ["null", "string"]},
+              "coupon": {
+                "type": ["null", "object"],
+                "properties": {
+                  "id": {"type": ["null", "object"]},
```

Comment:
Is it really have an `object` type? In your test logs you have following error message:
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/19872253/139603125-859fcdb8-0f9c-4cd8-9e23-96aa7dee00f0.png">

This means that in the schema file this field has an `object` type but in one of the records this field was `ANTELM` which is a `string` type. Could you please check this? You just need to run the `read` command and look at the record.